code/__HELPERS/game.dm

```diff
@@ -520,6 +520,12 @@
 		if(M && !isnewplayer(M))
 			to_chat(M, thing)
 
+/proc/sound_to_playing_players(sound, volume = 100, vary)
+	for(var/M in GLOB.player_list)
+		if(ismob(M) && !isnewplayer(M))
+			var/mob/MO = M
+			MO.playsound_local(MO, sound, volume, vary, pressure_affected = FALSE)

```

Comment:
It's mainly that it will silently fail to modify the sound in some cases, better to just use it properly